zmon_worker_monitor/builtins/plugins/scalyr.py

```diff
@@ -80,20 +81,29 @@ def facets(self, filter, field, max_count=5, minutes=30, prio='low'):
         j = r.json()
         return j
 
-    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low'):
-
+    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low', align=0):
+        startTime = str(minutes) + 'm'
+        endTime = None
+        if align != 0:
+            cur_time = int(time.time())  # this assumes the worker is running with UTC time
```

Comment:
I don't know the coding standards of ZMON projects, though three local vars (`cur_time`, `end`, `start`) seem to be redundant here..

Reply:
fixed :)